### DISCUSSION TITLE
OpenCollective email system not working, please use github instead?

### DISCUSSION THREAD

**Discussion by ProLoser** [parcel-bundler/parcel#5734] [Q&A]
I am trying to login to my opencollective account to support development of parcel2 but it's not working. I tried emailing their support but my email bounces. Do you think you guys could use the Github platform itself to collect donations instead? I'm not sure if anyone else has experienced this.

**Comment by devongovett** [comment]
cc @xdamman

**Comment by piamancini** [comment]
Hi @ProLoser I can help you out - We haven't heard about anyone else suffering this issue. What's your profile's slug on Open Collective?

**Reply by piamancini** [reply]
@ProLoser nm found it - I see one profile linked to proloser+opencollective@ account and one linked to proloser@ account. Are you trying to use one of these?

**Reply by piamancini** [reply]
Our logs indicate the login email went out from our servers - and shows as delivered on your end. I'll bring @znarf into this conversation to see if he can help out.

**Reply by ProLoser** [reply]
Is it possible to sponsor specific bugs being fixed on OpenCollective? There are some issues that block my project's development and would be happy to sponsor them getting fixed sooner.

**Reply by piamancini** [reply]
@ProLoser Sure! - can you open an issue here with the request: github.com/opencollective/opencollective? We have a bounty program so we are constantly putting out bounties for our community. We can look into your blockers for this or look into solving it ourselves. Please tag me if possible so I can make sure we see them.

**Reply by znarf** [reply]
@piamancini I confirm your findings, looks good from our side, the emails were delivered to Microsoft servers. Also, people seems to be reaching support reliably.

@ProLoser Not sure how to help at this point. If you have more info on the bounces, please let us know!

**Reply by znarf** [reply]
@ProLoser About bugs, are you talking about Parcel bugs or Open Collective bugs?

**Reply by ProLoser** [reply]
Parcel bugs

**Reply by piamancini** [reply]
ah - technically it's possible on Open Collective but it's up to @devongovett

**Reply by mischnic** [reply]
@ProLoser Out of interest, which issues are those? They might coincide with my own todo-list.

**Reply by ProLoser** [reply]
https://github.com/parcel-bundler/parcel/issues/1093 Our project is using audio worklets, and I was under the (mistaken?) impression they're fixed in `nightly` but now I'm not so sure. My custom `.parcelrc` has the following transformer `"rawjs:*": ["...", "@parcel/transformer-raw"]` we used as a solution to importing the audio worklet (transpiling and serving it but keeping it a separate file). This is kind of blocking my app from publishing.

https://github.com/parcel-bundler/parcel/issues/5724#issuecomment-769391337 I would love some clarification on this issue (not sure if my configuration was wrong or your comments were the culprit).

https://github.com/parcel-bundler/parcel/issues/4085#issuecomment-767241457 Support for class properties (or more specifically how to do a babel configuration with `shippedProposals:true`) would be nice, but I've since refactored all our react classes to be functions instead, so this isn't a priority anymore.

I'm not sure how many of these are related. Currently I can't bundle on our project, only parcel serve works

**Reply by ProLoser** [reply]
@piamancini I'm trying to use either or both of those emails, but neither one is ending up in my inbox for logging in. I've tried a dozen times

**Reply by ProLoser** [reply]
Yeah I still don't see any emails from opencollective and I'd definitely like to sponsor regularly if it helps the audioworklet bug get addressed. I'm currently trying to figure out how to get these things working with webpack as I don't understand the syntax detection logic.

**Comment by ProLoser** [comment]
Figured out what was going on.
